routers/lamoda_router.py

```diff
@@ -14,17 +14,9 @@ def get_products():
     Return all lamoda products in a database
     """
 
-    # example product - will be removed
-    product = ProductSchema(
-        category='home',
-        brand='Zara',
-        name='pillow',
-        price=230.1,
-        descroption={'color': 'red', 'material': 'cotton'}
-    )
-    products = database.products
-    products.insert_one(product.dict())
-    return paginate(products)
+    database = ContainerMongo().mongo
+    products = database.client.products
+    return ContainerMongo().get_list_lamoda_products()
```

Comment:
Use Depends() for injecting dependencies inside a coroutine

Reply:
Done